src/idb/idb_60/idb_60.json

```diff
@@ -0,0 +1,302 @@
+{
+    "name":"idb 60",
+    "vendorId":"0x494b",
+    "productId":"0x0001",
+    "lighting":"none",
+    "matrix":{
+        "rows":10,
+        "cols":8
+    },
+    "layouts":{
+        "labels":[
+            "Split Backspace",
+            "ISO Enter",
+            "Split Left Shift",
+            "Full Right Shift",
+            [
+                "Bottom Row",
+                "WKL",
+                "Tsangan",
+                "MX HHKB",
+                "True HHKB"
```

Comment:
Prefer an alternative to "Tsangan" (TL;DR it's an ambiguous term and a person's name) - I've been using "7U" and "7U WKL" for when it has blockers. You used "Blockerless" on the webpage, maybe use that?

Reply:
Sure, I wasn't aware about its ambiguity.

Reply:
Done